icd/api/pipeline_binary_cache.cpp

```diff
@@ -827,19 +844,17 @@ Util::IArchiveFile* PipelineBinaryCache::OpenReadOnlyArchive(
     VK_ASSERT(pFilePath != nullptr);
     VK_ASSERT(pFileName != nullptr);
 
+    Util::AllocCallbacks allocCallbacks = {m_pAllocationCallbacks,
+                                           vk::allocator::PalAllocFuncDelegator,
+                                           vk::allocator::PalFreeFuncDelegator};
+
```

Comment:
Why did you move this and rename it?  Keep the formatting with the braces that was there before.